Documentation/Maintainers/BackportChanges.rst

```diff
@@ -0,0 +1,72 @@
+.. include:: /Includes.txt
+
+.. _backport-changes:
+
+================
+Backport changes
+================
+
+Most of the time changes will be made to master and backported.
+
+When creating a PR, it is possible to add the Releases in the
+:ref:`commit message <general-conventions-commit-messages>`
+(as done in the core):
+
+.. code-block:: none
+
+   Releases: master, 10.4
+
+However, this is not enforced and it is rarely done, so it is
+up to the user merging a PR to backport the changes to previous
+versions.
+
+Here are some tips and conventions:
+
+**Who should backport?**
+
+* The user merging should take care of the backporting.
+
+**Should PR be backported and up to which version?**
+
+* Decisions about which versions to backport to are at
+  the discretion of this user doing the merging
+* It may be possible that in order to backport a change,
+  too many changes are necessary because of merge conflicts.
+  Sometimes, the structure has changed. In this case,
+  decide if it is worth the effort.
+
+**How to backport?**
+
+
+* If you cherry-pick a commit locally, you can use -x to
+  automatically insert information that this is a cherry-pick.
+  This makes the history clearer, sometimes minor changes
+  have to be done while backporting to fix merge conflicts.
+
+.. code-block::
+
+   git cherry-pick -x <commit-id>
+
+
+writes:
+
+.. code-block::
+
+   Author: Author <email>
+   Date:   Fri Jun 26 15:50:08 2020 +0200
+
+      [TASK] Subject
+
+      (cherry picked from commit 609493dd8893cbac7ad78aa38a23e02d011bb0c2)
+
+**How do we keep track of what is already backported?**
```

Comment:
As a side note: if the original PR is merged with a reference to the PR # in the commit, the cherry-picks will also be automatically be displayed in the PR which makes it even easier to see (for example https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-TSconfig/pull/128 )

Reply:
That looks great. How did you do that? (It looks like the commit message must be changed or is there an automatism that does that?)

I am ok if we change my original suggestions.

Reply:
When merging I usually chose "Squash and Merge" which squashes changes down to one commit and adds the original text and PR number to the commit headline. 
I then cherry-pick (without anything special) that one commit which gets exactly the same commit message with the issue reference -> pushing that adds that to the orginal PR.

Reply:
Ah, cool. I often use "Squash and merge" when there is more than one commit. I didn't notice that it adds the PR number to the commit headline.

I like this solution better than mine, will change my PR later.

Reply:
This also means the label "backporting done" which I have partially used is not necessary. I will remove that.

Reply:
If it is useful you can make "Squash and Merge" the default, see:

https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuring-commit-squashing-for-pull-requests